kuma/wiki/models.py

```diff
@@ -1431,6 +1432,13 @@ def get_full_url(self):
     def is_experiment(self):
         return self.slug.startswith(EXPERIMENT_TITLE_PREFIX)
 
+    @property
+    def is_namespace_page(self):
```

Comment:
A test is needed in ``test_models.py`` for this new function. The test in ``test_search`` would be a decent template for the test:

https://github.com/mozilla/kuma/blob/2ee1e66dec001d4ddf1c9fa0a32929d33faca39c/kuma/wiki/tests/test_search.py#L28-L35

If you aren't comfortable writing a test, @escattone or I can take that on.

Reply:
Thanks, John. I'm happy to write the test. I just ran out of time last night and figured that Travis would show me failing tests that I could then fix. I'll get something into models.py